microsoft-edge/extensions-chromium/getting-started/part2-content-scripts.md

```diff
@@ -5,23 +5,25 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 06/28/2022
+ms.date: 08/05/2022
 ---
 # Create an extension tutorial, part 2
 
-To see the completed extension package source for this part of the tutorial, go to [MicrosoftEdge-Extensions repo > extension-getting-started-part2](https://github.com/microsoft/MicrosoftEdge-Extensions/tree/main/Extension%20samples/extension-getting-started-part2/extension-getting-started-part2).  The source code has been updated from Manifest V2 to Manifest V3.
+To see the completed extension package source for this part of the tutorial, go to [MicrosoftEdge-Extensions repo > extension-getting-started-part2](https://github.com/microsoft/MicrosoftEdge-Extensions/tree/main/Extension%20samples/extension-getting-started-part2/extension-getting-started-part2).  
+
+The source code has been updated from Manifest V2 to Manifest V3.
 
 This tutorial covers the following extension technologies:
 *   Injecting JavaScript libraries into an extension.
 *   Exposing extension assets to browser tabs.
 *   Including content pages in existing browser tabs.
 *   Having content pages listen for messages from pop-ups and respond.
 
-You'll learn to update your pop-up menu to replace your static stars image with a title and a standard HTML button.  That button, when selected, passes that stars image, which is embedded in the extension, to the content page.  That image, is inserted into the active browser tab. Follow the below steps for further details.
+You'll learn to update your pop-up menu to replace your static stars image with a title and a standard HTML button.  That button, when selected, passes that star's image to the content page.  This image is now embedded in the extension and inserted into the active browser tab. Here are the steps.
```

Comment:
```suggestion
You'll learn to update your pop-up menu to replace your static stars image with a title and a standard HTML button.  That button, when selected, passes that image of stars to the content page.  This image is now embedded in the extension and inserted into the active browser tab. Here are the steps.
```
The image `stars.jpeg` is at bottom of rendered section:
https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/getting-started/part2-content-scripts?branch=pr-en-us-2098&tabs=v2#step-6-add-the-content-script-message-listener